tests/_root/shared/config/uploads.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
```

Comment:
This config with current defined mime types does not provide any real value since it's duplicates the cores default mime types, consider adding additional mimes